### PR TITLE
Controls/Media: Fix - Insert an image from URL (ED-2066)

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: assets/dev/js/editor/controls/media.js [hunk 1]
```diff
@@ -89,9 +89,9 @@ ControlMediaItemView = ControlMultipleBaseItemView.extend( {
 		// Set current doc id to attach uploaded images.
 		wp.media.view.settings.post.id = elementor.config.document.id;
 		this.frame = wp.media( {
-			button: {
-				text: __( 'Insert Media', 'elementor' ),
-			},
+			frame: 'post',
```
Comment (matipojo) [inline assets/dev/js/editor/controls/media.js:92]: Please add before/after screenshots.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/controls/media.js (hunk 1 above)
Comment (StyleShit) [reply assets/dev/js/editor/controls/media.js:92]: It looks the same.

Comment (matipojo) [reply assets/dev/js/editor/controls/media.js:92]: Even after removing unwanted elements from modal?

Comment (StyleShit) [reply assets/dev/js/editor/controls/media.js:92]: When you said "before/after", you meant specifically those lines you quoted or everything?

Comment (matipojo) [reply assets/dev/js/editor/controls/media.js:92]: Everything

**Pull request closed without merging**